lib/Wikia/src/Service/User/Preferences/UserPreferences.php

```diff
@@ -100,12 +95,16 @@ public function setMultiple( $userId, $prefs ) {
 		$this->save( $userId, $prefToSave );
 	}
 
+	/**
+	 * @param $pref
+	 * @return Optional
+	 */
 	public function getFromDefault($pref) {
```

Comment:
Is this function being used anywhere else? If so the caller may be broken since the return value changed.